FCModel/FCModel.m

```diff
@@ -163,6 +163,10 @@ + (instancetype)instanceWithPrimaryKey:(id)primaryKeyValue databaseRowValues:(NS
         // Not in memory yet. Check DB.
         instance = fieldValues ? [[self alloc] initWithFieldValues:fieldValues existsInDatabaseAlready:YES] : [self instanceFromDatabaseWithPrimaryKey:primaryKeyValue];
         if (! instance && create) {
+            if (!g_primaryKeyFieldName[self]) {
+                return nil;
```

Comment:
should this be `return (create ? [self new] : nil);` as it is for the `nil` check on `primaryKeyValue`?

Reply:
I don't think so, if the database doesn't exist we don't want to create anything.

Reply:
right 🤦‍♀️